content/master/concepts/packages.md

```diff
@@ -437,7 +437,7 @@ Including YAML files that aren't Compositions or CompositeResourceDefinitions,
 including Claims isn't supported.
 {{</hint >}}
 
-By default, Crossplane creates an `.xpkg` file of the Configuration name and 
+By default, Crossplane creates a `.xpkg` file of the Configuration name and 
```

Comment:
i didn't think this was right at first, but I guess it does actually work because it should be read as `creates a dot xpkg file`